### PR TITLE
Populate rest of about page

### PR DESCRIPTION
**Pull request by kevinjihwanlee**
The rest of the about page below the hero is now populated with content. This includes a basic description of what Serif.nu does as well as a Team section.

### CODE REVIEW

**Comment by kevinjihwanlee** [comment]
[Populate the rest of the About page](https://trello.com/c/GPClDXLZ/174-populate-the-rest-of-the-about-page)

File: src/components/pages/AboutPage.js [hunk 1]
```diff
@@ -39,6 +81,85 @@ function AboutPage({ classes }) {
...
+            <Typography variant="h5" className={classes.blurbs}>No more clunkly Excel spreadsheets.</Typography>
+            <Typography variant="h5" className={classes.blurbs}>Plan out your next quarter at Northwestern with ease and speed.</Typography>
+          </div>
+          <img src={studentImage} alt="Student with a laptop" className={classes.studentImage} />
+        </div>
+      </div>
+
+      <div className={classes.sectionWithColor}>
+        <Typography variant="h3" color="primary" className={classes.header}>
+          Features
```
Comment (Joonpark13) [inline src/components/pages/AboutPage.js]: Let's take this section out for now, as discussed

File: src/components/pages/AboutPage.js [hunk 2]
```diff
@@ -39,6 +81,85 @@ function AboutPage({ classes }) {
...
+      <div className={classes.section}>
+        <Typography variant="h3" color="primary" className={classes.header}>The Team</Typography>
+        <Typography variant="h5" className={classes.blurbs}>Serif.nu is run by a team of previous and current Northwestern students dedicated to providing the best scheduling experience.</Typography>
+        <div className={classes.multipleColumns}>
+          <Paper className={classes.studentCard}>
+            <img src={joonProfile} alt="Joon Park" className={classes.profileImage} />
+            <Typography variant="h6">Joon Park</Typography>
+            <Typography variant="subtitle1">Creator & Team Lead</Typography>
+            <Typography variant="body2" className={classes.blurbs}>Software engineer at Grubhub, film composer in another life. Big fan of taking pictures that make me look like I&apos;m falling off of things.</Typography>
+          </Paper>
```
Comment (Joonpark13) [inline src/components/pages/AboutPage.js:109]: Linebreak after each paper for readability?

File: src/components/pages/AboutPage.js [hunk 3]
```diff
@@ -39,6 +81,85 @@ function AboutPage({ classes }) {
...
+            <Typography variant="h5">Search for classes by name </Typography>
+            <Typography variant="h5">Browse by school and subject </Typography>
+            <Typography variant="h5">More coming soon!</Typography>
+          </div>
+        </div>
+      </div>
+
+      <div className={classes.section}>
+        <Typography variant="h3" color="primary" className={classes.header}>The Team</Typography>
+        <Typography variant="h5" className={classes.blurbs}>Serif.nu is run by a team of previous and current Northwestern students dedicated to providing the best scheduling experience.</Typography>
```
Comment (Joonpark13) [inline src/components/pages/AboutPage.js:102]: Linebreak here to delineate between header and the rest of section?

**Review by Joonpark13 — CHANGES_REQUESTED** [review]

File: src/components/pages/AboutPage.js (hunk 1 above)
Comment (kevinjihwanlee) [reply src/components/pages/AboutPage.js]: Removed.

File: src/components/pages/AboutPage.js (hunk 2 above)
Comment (kevinjihwanlee) [reply src/components/pages/AboutPage.js:109]: Done

File: src/components/pages/AboutPage.js (hunk 3 above)
Comment (kevinjihwanlee) [reply src/components/pages/AboutPage.js:102]: Done

**Review by Joonpark13 — APPROVED** [review]
Looks great!

**Pull request merged**